### PR TITLE
Improve migration logging a bit

### PR DESCRIPTION
**Pull request by tas50**
This makes it more clear when running the upgrade that we're talking
about Database migrations.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/2988#e83736e0-28bf-4cb5-a348-9c543861b8bc

File: omnibus/partybus/bin/partybus [hunk 1]
```diff
@@ -140,18 +140,18 @@ end
...
   pending_migrations = migrations.select{ |m| m > migration_state }.sort
-  log("Latest Migration Available: #{migrations.last}")
-  log("Migrations to Run: #{pending_migrations}")
+  log("Latest Database Migration Available: #{migrations.last}")
+  log("Database Migrations to Run: #{pending_migrations}")
 
   # run them through the DSL
   pending_migrations.each do |migration|
-    log("Current Migration Version: #{migration_state}")
+    log("Current Database Migration Version: #{migration_state}")
```
Comment (PrajaktaPurohit) [inline omnibus/partybus/bin/partybus:148]: should we be specific about this and say PostgreSQL?

**Review by PrajaktaPurohit — APPROVED** [review]
Left a question, also ok to merge as is.

File: omnibus/partybus/bin/partybus (hunk 1 above)
Comment (tas50) [reply omnibus/partybus/bin/partybus]: I updated all of these

**Pull request merged**